src/sentences/gsa.rs

```diff
@@ -85,7 +90,35 @@ fn gsa_prn_fields_parse(i: &str) -> IResult<&str, Vec<Option<u32>, 18>> {
     many0(terminated(opt(number::<u32>), char(',')))(i)
 }
 
-type GsaTail = (Vec<Option<u32>, 18>, Option<f32>, Option<f32>, Option<f32>);
+type GsaTail = (
+    Vec<Option<u32>, 18>,
+    Option<f32>,
+    Option<f32>,
+    Option<f32>,
+    Option<GnssType>,
+);
+
+fn do_parse_gsa_tail_with_gnss_type(i: &str) -> IResult<&str, GsaTail> {
+    let (i, prns) = gsa_prn_fields_parse(i)?;
+    let (i, pdop) = float(i)?;
+    let (i, _) = char(',')(i)?;
+    let (i, hdop) = float(i)?;
+    let (i, _) = char(',')(i)?;
+    let (i, vdop) = float(i)?;
+    let (i, _) = char(',')(i)?;
+    let (i, gnss_type) = number::<u8>(i)?;
```

Comment:
If I understand correctly this is the System ID:
https://gpsd.gitlab.io/gpsd/NMEA.html#_gsa_gps_dop_and_active_satellites

At least from this documentation I don't see 5 and 6 but if your receiver uses these numbers I'm ok with including them.